deploy/openshift/templates/monitoring/grafana-dashboards.yaml

```diff
@@ -6241,6 +6241,581 @@ data:
       "uid": "sL0Klq_mz",
       "version": 1
     }
+    che-tomcat.json: |-
```

Comment:
can you add panels to Che Server dashboard?

Reply:
ok